test/spec/basic.rb

```diff
@@ -0,0 +1,129 @@
+# Copyright (c) 2016 Chef Software Inc. and/or applicable contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'pathname'
+
+# TODO: .rspec file doesn't seem to be honored, so we need
+# to manually include the spec_helper here
+require_relative 'spec_helper'
+
+describe "Habitat CLI" do
+    # TODO: maybe extract this into a module in the future
+    before(:all) do
```

Comment:
Ruby's indentation standard is 2 spaces. Forgive me for including stylistic comments in a PR
![gif-keyboard-1300383516384764049](https://cloud.githubusercontent.com/assets/54036/17493267/6ae6d99e-5d64-11e6-8b7e-4eb5559f1d15.gif)